### ISSUE TITLE
train video with custom dataset

### ISSUE THREAD

**Issue by abdou31** [AlexeyAB/darknet#2785]
I trying to detect object in video , I'm using my own custom dataset
My questions is ,
+ Is it enough to train datasets of images to detect video and streaming ( webcam )?
+ When should i stop the training ( number of iterations ) to get good resultat? 

Thanks,

**Comment by AlexeyAB** [comment]
* If you will detect only from the same camera from the same position, then you can train your model on images from this camera.

* train more than `max(4000, 2 * classes)` iterations or while mAP increases: https://github.com/AlexeyAB/darknet#when-should-i-stop-training

**Comment by abdou31** [comment]
Thanks , 
I have one classe for now ,
I have other question , I want to align object to some reference template. like ( faces for exemple ) , 
Like this : 
https://raw.githubusercontent.com/1adrianb/face-alignment/master/docs/images/face-alignment-adrian.gif
I want to know is it possible to do that with Darknet or no ?
What files should be modified to algin object ?

**Comment by AlexeyAB** [comment]
No, Darknet doesn't support keypoints (facial-landmarks, facial-keypoints, facial-features).

**Comment by abdou31** [comment]
+ What about drawing circle?

**Comment by AlexeyAB** [comment]
@abdou31 You can try this: https://github.com/AlexeyAB/darknet/issues/2725

**Comment by abdou31** [comment]
Thanks ,
but you don't give the function of drawing circle in image, you give just the implementation of function draw_box in file image.c
+ How to implement function draw_circle(x,y,r) ?
+ How to solve the problem of values of yolo ? 
( darknet request 4 variables for annotation , but circle just have 3 arguments x , y of the center and r the raduis
